### PR TITLE
fix syntax in readme

### PR DESCRIPTION
**Pull request by wongoo**
**What this PR does**:
fix syntax in readme

**Which issue(s) this PR fixes**:
NONE

**Special notes for your reviewer**:
NONE

**Does this PR introduce a user-facing change?**:
NONE

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,8 +51,9 @@ circular, ok := obj.(*Circular)
 
 ## Customize Usage Examples
 
-#### Struct filed name encoding
-Hessian encoder will convert the filed name of struct to lower camelcase defaulted, but you customize it by `hessian` tag of struct.
+#### Encoding filed name 
+
+Hessian encoder default converts filed names of struct to lower camelcase, but you can customize it using `hessian` tag.
```
Comment (AlexStocks) [inline README.md]: but you can customize it by using `hessian` tag. or but you can customize it by `hessian` tag.

File: README.md [hunk 2]
```diff
@@ -76,7 +77,8 @@ if err != nil {
     panic(err)
 }
 ``` 
-The encoded bytes of the struct from above example will look like:
+
+The encoded bytes of the struct in the example above:
```
Comment (AlexStocks) [inline README.md]: The encoded bytes of the struct MyUser is as following:

File: README.md [hunk 3]
```diff
@@ -85,8 +87,11 @@ The encoded bytes of the struct from above example will look like:
  00000040  0c 30 31 30 2d 31 32 33  34 35 36 37 38           |.010-12345678|
 ```
 
-#### Struct filed name decoding
-Hessian decoder will compare all filed's name of struct until it matches the correct name, the order of matching rules is:
+#### Decoding filed name 
+
+Hessian decoder find the correct target field though comparing all filed names of struct one by one until matching.
```
Comment (AlexStocks) [inline README.md]: Hessian decoder finds the correct target field through comparing all filed names of struct one by one until matching.

File: README.md [hunk 4]
```diff
@@ -85,8 +87,11 @@ The encoded bytes of the struct from above example will look like:
...
+
+The following example show the order of the matching rules:
```
Comment (AlexStocks) [inline README.md]: The following example shows the order of the matching rules:

File: README.md [hunk 5]
```diff
@@ -159,7 +164,8 @@ if err != nil {
     panic(err)
 }
 ``` 
-The encoded bytes of the struct from above example will look like:
+
+The encoded bytes of the struct in the example above:
```
Comment (AlexStocks) [inline README.md]: The encoded bytes of the struct MyUser is as following:

File: README.md (hunk 1 above)
Comment (wongoo) [reply README.md]: the syntax `customize something using something` is also correct.

File: README.md (hunk 2 above)
Comment (wongoo) [reply README.md]: should be `as follow`

File: README.md [hunk 6]
```diff
@@ -78,7 +78,7 @@ if err != nil {
 }
 ``` 
 
-The encoded bytes of the struct in the example above:
+The encoded bytes of the struct `MyUser` is as follow:
```
Comment (AlexStocks) [inline README.md]: 如果你用 as follow，后面加上 逗号。如果用冒号，就用 following

File: README.md [hunk 7]
```diff
@@ -165,7 +165,7 @@ if err != nil {
...
```
Comment (AlexStocks) [inline README.md]: 同上

**Pull request merged**